packages/storybook-addon-markdown-docs/src/mdToJsx.js

```diff
@@ -48,7 +48,7 @@ __export_default__.parameters = __export_default__.parameters || {};
 __export_default__.parameters.docs = __export_default__.parameters.docs || {};
 __export_default__.parameters.docs.page = () => <AddContext
   mdxStoryNameToKey={mdxStoryNameToKey}
-  mdxComponentMeta={__export_default__}><MDXContent
+  mdxComponentAnnotations={__export_default__}><MDXContent
```

Comment:
The MDJS Stories were failing in the docs page with the previous value. This is what the MDX files were delivering, so I just normalized the two. Not sure how to fully confirm this is "correct" beyond that.